netbsd-from-tarball CI fails during config

Example: https://github.com/transmission/transmission/actions/runs/22023861561/job/63637324661

```
2026-02-14T20:39:09.4273436Z -- Checking for modules 'gtkmm-4.0>=4.11.1;glibmm-2.68>=2.60.0;giomm-2.68>=2.26.0'
2026-02-14T20:39:09.5410366Z --   Package 'xrender', required by 'pango', not found
2026-02-14T20:39:09.5410961Z Package 'xproto', required by 'xft', not found
2026-02-14T20:39:09.5411263Z Package 'x11', required by 'cairo', not found
2026-02-14T20:39:09.5411545Z Package 'xext', required by 'cairo', not found
2026-02-14T20:39:09.5411869Z Package 'xrender', required by 'cairo', not found
2026-02-14T20:39:09.5412158Z Package 'xcb', required by 'cairo', not found
2026-02-14T20:39:09.5412460Z Package 'xcb-render', required by 'cairo', not found
2026-02-14T20:39:09.5412760Z Package 'xcb-shm', required by 'cairo', not found
2026-02-14T20:39:09.5413062Z Package 'xrender', required by 'pangoft2', not found
2026-02-14T20:39:09.5413391Z Package 'xrender', required by 'pangocairo', not found
2026-02-14T20:39:09.5413732Z Package 'x11', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5414051Z Package 'xext', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5414381Z Package 'xrender', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5414834Z Package 'xcb', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5415354Z Package 'xcb-render', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5415919Z Package 'xcb-shm', required by 'cairo-gobject', not found
2026-02-14T20:39:09.5485304Z -- Checking for modules 'gtkmm-3.0>=3.24.0;glibmm-2.4>=2.60.0;giomm-2.4>=2.26.0'
2026-02-14T20:39:09.5569719Z --   Package 'gtkmm-3.0' not found
2026-02-14T20:39:09.5649980Z --   Package 'glibmm-2.4' not found
2026-02-14T20:39:09.5741447Z --   Package 'giomm-2.4' not found
2026-02-14T20:39:09.5747679Z CMake Error at CMakeLists.txt:356 (message):
2026-02-14T20:39:09.5748268Z   GTK is required but wasn't found
2026-02-14T20:39:09.5748452Z 
2026-02-14T20:39:09.5748456Z 
2026-02-14T20:39:09.5754283Z -- Configuring incomplete, errors occurred!
2026-02-14T20:39:09.5851391Z ##[endgroup]
2026-02-14T20:39:09.5880911Z ##[error]The process '/usr/bin/ssh' failed with exit code 1
```

Related to #7901 and likely #8459

--- a/.github/workflows/actions.yml
+++ b/.github/workflows/actions.yml
@@ -1439,7 +1439,7 @@ jobs:
               `[ '${{ needs.what-to-make.outputs.make-qt }}' = 'true' ] && echo qt6-qtsvg qt6-qttools` \
               rapidjson \
               utf8-cpp
-            if [ '${{ needs.what-to-make.outputs.make-qt }}' = 'true' ]; then
+            if [ '${{ needs.what-to-make.outputs.make-gtk }}' = 'true' ] || [ '${{ needs.what-to-make.outputs.make-qt }}' = 'true' ]; then
               for set in xbase xcomp; do
                 curl -sL "https://cdn.netbsd.org/pub/NetBSD/NetBSD-`uname -r`/${{ matrix.arch.binary-set }}/binary/sets/$set.tar.xz" | tar xJpf - -C /
               done

